src/VisualStudio/Core/Def/Implementation/Venus/VenusCommandFilter.cs

```diff
@@ -35,7 +36,18 @@ internal class VenusCommandFilter<TPackage, TLanguageService> : AbstractVsTextVi
 
             _subjectBuffer = subjectBuffer;
             CurrentHandlers = commandHandlerServiceFactory.GetService(subjectBuffer);
-            NextCommandTarget = nextCommandTarget;
+            // Setup all command handlers migrated to the modern editor commandig to be execited next
+            var componentModel = Shell.ServiceProvider.GlobalProvider.GetService(typeof(SComponentModel)) as IComponentModel;
```

Comment:
[mop-up in Preview 4] We generally avoid GlobalProvider since it has potenetial surprises. You should have a Package type here you can grab it from. Also we hard cast because if that as ... doesn't work, you're in deep trouble. :smile: